packages/excerpter/lib/src/inject.dart

```diff
@@ -4,6 +4,8 @@
 import 'dart:io';
 
 import 'package:collection/collection.dart';
+import 'package:deviation/deviation.dart';
```

Comment:
Is this package new?

Reply:
Yeah I made it, but it has a few issues still, so I'm keeping this support as experimental until I can iron them out.